packages/core/src/components/ColorPalette/ColorDisplay.tsx

```diff
@@ -124,7 +120,28 @@ export function ColorDisplay({
         mb={5}
         style={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}
       >
-        <Text style={{ fontWeight: 600, fontSize: '1.5rem' }}>{colorName}</Text>
+        <Input
```

Comment:
I think the expected behaviour should be that the input box opens up only after double-clicking on the color name ( rather than single-click, which is the behaviour in this PR ). Thoughts @alexdanilowicz ?

Reply:
Agreed @Pranav2612000.